networks/suzuka/suzuka-client/src/tests/mod.rs

```diff
@@ -130,5 +131,29 @@ async fn test_example_interaction() -> Result<()> {
 			.context("Failed to get Bob's account balance the second time")?
 	);
 
+	sleep(Duration::from_secs(10)).await;
+
+	let anvil_rpc_port = "8545";
+	let anvil_rpc_url = format!("http://localhost:{anvil_rpc_port}");
+	let anvil_ws_url = format!("ws://localhost:{anvil_rpc_port}");
```

Comment:
I'm not sure if I'm comfortable with adding "unit" tests that require additional infra to run. I'd like `cargo test` in the root of the checked out workspace to not require any external setup. But this may be too late to complain about this if we have similar requirements for other crates' tests already.

I think such e2e-ish tests should be in their own crate, or perhaps they should be feature-gated to not be  run by `cargo test` by default.

Reply:
Looks like this will be addressed in its own PR. But yes, agree.